.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -83,6 +83,8 @@ body:
           required: true
         - label: I have chosen an appropriate title.
           required: true
+        - label: I have used supported version, e.g. YouTube v 19.16.39.
+         required: true
```

This won't necessarily make users using versions other than 19.16.39 realize that the version they are using is not supported, since 19.16.39 is only an example. It also is better to avoid using a static version number in the acknowledgment since it will need to manually be changed in the future.

Here is my suggestion (also i reordered the labels to keep similar ones near each other)

```suggestion
        - label: The version of the patched APK I am using is currently supported, as per the [list of supported versions](https://github.com/inotia00/revanced-patches?tab=readme-ov-file#-json-format).
         required: true
        - label: I have chosen an appropriate title.
          required: true
```